examples/camera-app/linux/include/clusters/camera-avstream-mgmt/camera-av-stream-manager.h

```diff
@@ -32,7 +29,7 @@ namespace CameraAvStreamManagement {
 /**
  * The application delegate to define the options & implement commands.
  */
-class CameraAVStreamManager : public CameraAVStreamMgmtDelegate
+class CameraAVStreamManager : public CameraAVStreamController
```

Comment:
The CameraAVStreamManager can implement the CameraAVStreamMgmtDelegate and the CameraAVStreamController interface separately?
That way, the other classes can stay the same, i.e., only implementing their Delegate.